aries-site/src/pages/index.js

```diff
@@ -29,7 +30,7 @@ const Index = () => {
     }
   }, []);
 
-  const data = Object.values(homeData);
+  const data = structure.filter(page => page.pages);
```

Comment:
What happens if a content page gets populated with a `pages` property?  Won't it now errantly get picked up in this filter?

What if we gave the object in the data structure a property of `parent: 'Home'` and the modified the filter to be `const data = structure.filter(page => page.parent === title` (Where title is a variable that then populates the `<PageLayout>` title as well)?   This way, we will only receive pages for which we are explicitly asking.

Reply:
I like this. I'll add that in.